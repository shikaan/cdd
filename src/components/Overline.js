import styled from "styled-components";

const Overline = styled.span(({ theme }) => `
  display: block;
  font-weight: 500;
  font-family: ${theme.typography.secondaryFont.fontFamily};
  font-size: ${theme.typography.baseFontSize.multiply(0.75)};
  padding-bottom: ${theme.typography.baseFontSize.multiply(0.5)};
  color: ${theme.color.dark600};
`);

export default Overline;
