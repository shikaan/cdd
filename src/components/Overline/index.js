import styled from 'styled-components'

export default styled.span(({ theme }) => `
  display: block;
  font-weight: 500;
  font-family: ${theme.typography.secondaryFont.fontFamily};
  font-size: ${theme.typography.baseFontSize.multiply(0.75)}
  padding-bottom: ${theme.typography.baseFontSize.multiply(0.5)};
  color: ${theme.color.black};
`)
