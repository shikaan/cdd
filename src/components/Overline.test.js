import React from 'react'
import {shallow} from 'enzyme';
import 'jest-styled-components';

import Overline from "./Overline";

describe('Overline', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Overline/>)

    expect(wrapper).toMatchSnapshot()
  })
})
