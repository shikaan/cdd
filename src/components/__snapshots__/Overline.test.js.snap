// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overline renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          [Function],
        ],
      },
      "displayName": "styled.span",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "span",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
